openapi/platform/platform.yaml

```
@@ -3289,6 +3289,52 @@ paths:
       tags:
         - Options
         - User Custom Fields
+  '/users/{id}/actions/sendWelcomeEmail':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+    get:
+      summary: Send Welcome Email
+      operationId: get-sendWelcomeEmail-by-userid
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        This endpoint works similar to the resend welcome email option available in the UI of the partner center. An email will be send to the user whose user id is passed in the request. 
```

```suggestion
        An email will be sent to the already created user whose user id is passed in the request. 
```

imo, we don't have to mention anything about UI, if we mention, we got to elaborate to the new users? :)

---

Yes you are right. We dont have to mention that